File: TiaCodegen/Commands/Coils/PCoil.cs
using TiaCodegen.Commands.Signals;
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands.Coils
{
    public class PCoil : BaseNPCoil
    {
        public PCoil(Signal signal, Signal helpSignal, IOperationOrSignal op = null) : base(signal, helpSignal, op)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/Coils/PCoil.ts
import { IOperationOrSignal } from '../../Interfaces/IOperationOrSignal';
import { Signal } from '../Signals/Signal';
import { BaseNPCoil } from './BaseNPCoil';

export class PCoil extends BaseNPCoil {
    constructor(signal: Signal, helpSignal: Signal, op: IOperationOrSignal | null = null) {
        super(signal, helpSignal, op);
    }
}
